src/actions/profile-view.js

```diff
@@ -85,6 +85,11 @@ export function changeThreadOrder(threadOrder: ThreadIndex[]): Action {
   };
 }
 
+function dispatchResizeEvent() {
+  const event = new UIEvent('resize', { view: window });
+  window.dispatchEvent(event);
+}
+
```

Comment:
I don't feel comfortable dispatching DOM events in redux thunk actions. Moreover it's not guaranteed that the React Components are actually rerendered -- maybe they're rendered currently but it might not be the case in a future version, because they're making React more and more asynchronous.

My suggestion would be to dispatch the event in `ProfileViewerHeader` instead, in a `componentDidUpdate` method. Because `componentDidUpdate` gets passed the `prevProps`, we can check whether the prop `hiddenThreads` has changed, and dispatch the event if that's the case.

I think this makes it better because:
* the event is dispatched in one place only, we don't need to track all the places where we dispatch an action that changes the state
* it's guaranteed that the React component is rendered
* the React component is the proper place for side effects.

@gregtatum I know you suggested this code initially, please tell me what you think?

Reply:
just discussed with @gregtatum, let's move forward with what I suggest here.
Sorry again for making you changing this...